hackathon_site/hardware/test_api.py

```diff
@@ -954,7 +963,9 @@ def test_user_has_no_profile(self):
 
     def test_create_simple_order(self):
         self._login()
-        profile = self._make_event_profile()
+        self.profile = Profile.objects.create(user=self.user, team=self.team)
```

Comment:
Nice job on the tests!
Looking at your changes, you're repeating creating 2 profiles in almost all the tests. Since this is repeated code, you can put it in a helper function in this class just to create 2 profiles.

under `setUp()`
```
def create_min_number_of_profiles(self):
   Profile.objects.create(user=self.user, team=self.team)
   Profile.objects.create(user=self.user2, team=self.team)
```
and call that function in your tests. Since you don't use `self.profile` or `self.profile2` anywhere, you don't need to assign the profiles to a variable.